recipes/svim/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version="0.3.1" %}
+
+package:
+  name: svim
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ba/1b/80a7542b27637ee85ff9239b10ac14283be3c2fcf52a9659d3cc82b34929/svim-{{ version }}.tar.gz
+  sha256: bf4aae8e774bb2621d8ad649148152a73ad07952a04b672aaf6201546b900d76
+
+build:
+  number: 0
+  skip: True # [py27]
+
+requirements:
+  host:
+    - python 3.6
```

Comment:
Make this `3.6.*` - patch releases should be ok

Reply:
Thanks, done.